example/examples/AnimatedMarkers.js

```diff
@@ -62,18 +62,28 @@ const AnimatedMarkers = React.createClass({
         </View>
       </View>
     );
-  },
-});
+  }
+}
+
 
 
-let styles = StyleSheet.create({
+const styles = StyleSheet.create({
   container: {
     ...StyleSheet.absoluteFillObject,
+    position: 'absolute',
```

Comment:
Remove these 5 styles.  This is what `...StyleSheet.absoluteFillObject` provides.